editor/store/reducer.js

```diff
@@ -661,60 +661,55 @@ const locations = [
 const defaultMetaBoxState = locations.reduce( ( result, key ) => {
 	result[ key ] = {
 		isActive: false,
-		isDirty: false,
-		isUpdating: false,
 	};
 
 	return result;
 }, {} );
 
+/**
+ * Reducer keeping track of the meta boxes saving state.
```

Comment:
Personally I don't find this comment to be particularly helpful beyond pointing out that it's a reducer, and would encourage elaborating on the purpose for why we track a meta box saving state.